raft/rafttest/node.go

```diff
@@ -49,11 +50,15 @@ func (n *node) start() {
 					n.storage.SetHardState(n.state)
 				}
 				n.storage.Append(rd.Entries)
+				var wg sync.WaitGroup
+				wg.Add(1)
 				go func() {
+					defer wg.Done()
 					for _, m := range rd.Messages {
 						n.iface.send(m)
 					}
 				}()
+				wg.Wait()
```

Comment:
Why even both with the goroutine if we're going to wait on it (and nothing else) anyway? Just move the for loop up to this level.